### PR TITLE
Added descriptions about slow node detection

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Added descriptions about slow node detection

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/6831 https://github.com/pingcap/docs-cn/pull/6871
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -637,6 +637,13 @@ Configuration items related to Raftstore
 + Default value: `1`
 + Minimum value: greater than `0`
 
+### `inspect-interval`
+
++ At a certain interval, TiKV inspects the latency status of the Raftstore thread. This parameter specifies the interval of inspection. If the latency exceeds this value, the Raftstore thread is marked as timeout.
```
Comment (5kbpers) [inline tikv-configuration-file.md:642]:
```suggestion
+ At a certain interval, TiKV inspects the latency of the Raftstore component. This parameter specifies the interval of inspection. If the latency exceeds this value, the Raftstore thread is marked as timeout. 
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -637,6 +637,13 @@ Configuration items related to Raftstore
 + Default value: `1`
 + Minimum value: greater than `0`
 
+### `inspect-interval`
+
++ At a certain interval, TiKV inspects the latency status of the Raftstore thread. This parameter specifies the interval of inspection. If the latency exceeds this value, the Raftstore thread is marked as timeout. 
++ Judges whether the TiKV node is slow based on the ratio of inspected latency.
```
Comment (5kbpers) [inline tikv-configuration-file.md:643]:
```suggestion
+ Judges whether the TiKV node is slow based on the ratio of timeout inspection. 
```

File: pd-control.md [hunk 3]
```diff
@@ -701,6 +701,7 @@ Usage:
 >> scheduler config evict-leader-scheduler        // Display the stores in which the scheduler is located since v4.0.0
 >> scheduler add shuffle-leader-scheduler         // Randomly exchange the leader on different stores
 >> scheduler add shuffle-region-scheduler         // Randomly scheduling the regions on different stores
+>> scheduler add evict-slow-store-scheduler       // When there is one and only one slow score, evict all region leaders of that score
```
Comment (TomShawn) [inline pd-control.md:704]:
```suggestion
>> scheduler add shuffle-region-scheduler         // Randomly scheduling the Regions on different stores
>> scheduler add evict-slow-store-scheduler       // When there is one and only one slow score, evict all Region leaders of that score
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -637,6 +637,13 @@ Configuration items related to Raftstore
 + Default value: `1`
 + Minimum value: greater than `0`
 
+### `inspect-interval`
+
++ At a certain interval, TiKV inspects the latency of the Raftstore component. This parameter specifies the interval of inspection. If the latency exceeds this value, the Raftstore thread is marked as timeout.
```
Comment (TomShawn) [inline tikv-configuration-file.md:642]:
```suggestion
+ At a certain interval, TiKV inspects the latency of the Raftstore thread. This parameter specifies the interval of the inspection. If the latency exceeds this value, the Raftstore thread is marked as timeout. 
```

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -637,6 +637,13 @@ Configuration items related to Raftstore
 + Default value: `1`
 + Minimum value: greater than `0`
 
+### `inspect-interval`
+
++ At a certain interval, TiKV inspects the latency of the Raftstore component. This parameter specifies the interval of inspection. If the latency exceeds this value, the Raftstore thread is marked as timeout. 
++ Judges whether the TiKV node is slow based on the ratio of timeout inspection. 
++ Default value: 500ms
++ Minimum value: 1ms
```
Comment (TomShawn) [inline tikv-configuration-file.md:645]:
```suggestion
+ Default value: `"500ms"`
+ Minimum value: `"1ms"`
```

**Review by TomShawn — APPROVED** [review]
Good translation!
These are only very minor comments.

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
@5kbpers PTAL again. Thanks!

File: pd-control.md [hunk 6]
```diff
@@ -700,7 +700,8 @@ Usage:
 >> scheduler add evict-leader-scheduler 1         // Move all the Region leaders on store 1 out
 >> scheduler config evict-leader-scheduler        // Display the stores in which the scheduler is located since v4.0.0
 >> scheduler add shuffle-leader-scheduler         // Randomly exchange the leader on different stores
->> scheduler add shuffle-region-scheduler         // Randomly scheduling the regions on different stores
+>> scheduler add shuffle-region-scheduler         // Randomly scheduling the Regions on different stores
+>> scheduler add evict-slow-store-scheduler       // When there is one and only one slow score, evict all Region leaders of that score
```
Comment (5kbpers) [inline pd-control.md:704]:
```suggestion
>> scheduler add evict-slow-store-scheduler       // When there is one and only one slow store, evict all Region leaders of that store
```

File: tikv-configuration-file.md [hunk 7]
```diff
@@ -676,6 +676,13 @@ Configuration items related to Raftstore
 + Controls whether to enable batch processing of the requests. When it is enabled, the write performance is significantly improved. 
 + Default value: `true`
 
+### `inspect-interval`
+
++ At a certain interval, TiKV inspects the latency of the Raftstore thread. This parameter specifies the interval of the inspection. If the latency exceeds this value, the Raftstore thread is marked as timeout.
```
Comment (5kbpers) [inline tikv-configuration-file.md:681]:
```suggestion
+ At a certain interval, TiKV inspects the latency of the Raftstore component. This parameter specifies the interval of the inspection. If the latency exceeds this value, that inspection is marked as timeout. 
```

Comment (TomShawn) [reply tikv-configuration-file.md:681]: 中文里用的是 Raftstore 线程

Comment (5kbpers) [reply tikv-configuration-file.md:681]: 正确的描述还是还是 inspection，感觉中文的需要改一下，merge comment 的时候没有注意到这个地方

Comment (TomShawn) [reply tikv-configuration-file.md:681]: 中文已在 https://github.com/pingcap/docs-cn/pull/6908 中修改。PTAL

File: tikv-configuration-file.md [hunk 8]
```diff
@@ -676,6 +676,13 @@ Configuration items related to Raftstore
 + Controls whether to enable batch processing of the requests. When it is enabled, the write performance is significantly improved. 
 + Default value: `true`
 
+### `inspect-interval`
+
++ At a certain interval, TiKV inspects the latency of the Raftstore component. This parameter specifies the interval of the inspection. If the latency exceeds this value, that inspection is marked as timeout.
```
Comment (TomShawn) [inline tikv-configuration-file.md:681]:
```suggestion
+ At a certain interval, TiKV inspects the latency of the Raftstore component. This parameter specifies the interval of the inspection. If the latency exceeds this value, this inspection is marked as timeout. 
```

**Review by 5kbpers — APPROVED** [review]

**Pull request merged**